js/plugins/iq.js

```diff
@@ -236,7 +246,29 @@ var IqPlugin= function (framework)
         this.ooyalaReporter.reportReplay();
         OO.log("IQ: Reported: reportReplay()");
         break;
-
+      //OO.EVENTS.BUFFERING -> OO.Analytics.EVENTS.VIDEO_BUFFERING_STARTED.
+      case OO.Analytics.EVENTS.VIDEO_BUFFERING_STARTED: 
+        eventParams = params[0];
```

Comment:
Can we add a null check to see if params and params[0] exists?

Reply:
Added